salttesting/jenkins.py

```diff
@@ -245,21 +247,45 @@ def build_pillar_data(options, convert_to_yaml=True):
     if options.test_git_commit is not None:
         pillar['test_git_commit'] = pillar['repo_clone_rev'] = options.test_git_commit
     if options.test_git_url is not None:
-        pillar['test_git_url'] = pillar['repo_clone_url'] = options.test_git_url
+        # Due to quoting issues in Linux shelling out to Windows using WinEXE
+        # it is necessary to wrap some parameters in some crazy quotes '"'"'
+        # But, it's being converted to YAML at the end, so I don't know what's
+        # going to happen.
+        if getattr(options, 'windows', False):
```

Comment:
I forgot this was being initialized in the parser. I will change them back to `options.windows`